src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions.mdx

```diff
@@ -66,7 +66,7 @@ These permissions pertain to our [legacy alerting feature](/docs/alerts-applied-
 Permissions: 
 * **Channels**: relates to [alert notification channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts).
 * **Conditions**: relates to [alert conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions).
-* **Incidents**: relates to [alert incidents](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents) and [view incidents from our products](/docs/alerts-applied-intelligence/new-relic-alerts/alert-violations/view-alert-violations-our-products).
```

Comment:
looks like the view incidents from our products URL needs a redirect

Reply:
I'm going to add it just in case, but I've already replaced all the URLs, so there is no link to that page in the entire doc site.